src/test/java/org/folio/rest/api/StorageTestSuite.java

```diff
@@ -268,6 +268,7 @@ static void prepareTenant(String tenantId, boolean loadSample)
     assertThat(failureMessage,
       response.getStatusCode(), is(201));
 
+    tenantPrepared = new CompletableFuture<>();
```

Comment:
Thank you for spotting this, it might answer the issues that @nielserik was having a while ago with inconsistencies in loading records.

Rather than declare a new future for the same variable, could we instead introduce a new variable for the second step, maybe called {{tenantUpgraded}}, as that seems to reflect the intent of what is happening?

Reply:
@julianladisch What do you think about this proposal?